nntrainer/include/bn_layer.h

```diff
@@ -121,10 +121,25 @@ class BatchNormalizationLayer : public Layer {
 private:
   Tensor weight;
   Tensor bias;
+  /**
+   * @brief moving mu used for inferencing.
+   *        momentum * mu + (1 - momenutm) * mu of current batch is used
+   */
```

Comment:
No. you usually don't do this for class/struct properties. 

https://www.doxygen.nl/manual/docblocks.html
> int var; /**< Detailed description after the member */

or
> int var; ///< Detailed description after the member */

Reply:
Thank you for pointing this out! One question though, is this rule applied also for comments inside a function?